main.go

```diff
@@ -68,14 +70,14 @@ func run() error {
 								Usage:     "generate OpenAPI JSON from JSight",
 								Flags:     []cli.Flag{},
 								ArgsUsage: "<input>",
-								Action:    convert(converter.FormatJSON),
+								Action:    makeResult(format.FormatJSON),
 							},
 							{
 								Name:      "yaml",
 								Usage:     "generate OpenAPI YAML from JSight",
```

Comment:
It might be better to write: converts JSight to OpenAPI YAML

Reply:
fixed